test/lib/system_helpers_test.rb

```diff
@@ -60,5 +60,25 @@ module ForemanMaintain
         refute system.check_max_version('katello', '3.18')
       end
     end
+
+    describe 'format_shell_args options' do
+      it 'returns the string without single or double quotes' do
+        config = { 'user' => 'foo', 'password' => 'foopassword1!' }
+        escaped_string = ' user foo password foopassword1\\!'
```

Comment:
I wonder if we also should include `'` and/or `"` in the to-be-escaped strings somewhere, but on the other hand we're not writing tests for `Shellwords.escape` here…